### PR TITLE
Implemented messenger in order to send notifications via Sms and Email

### PR DESCRIPTION
**Pull request by DenisCrasnean**

### CODE REVIEW

**Comment by ovidiuenache** [comment]
@DenisCrasnean the code review process starts if the pipeline passes. In this case, it does not.

File: config/packages/framework.yaml [hunk 1]
```diff
@@ -29,17 +29,33 @@ framework:
...
 
+            smsNotification_client:
+                headers:
+                    X-API-KEY: '%env(SMS_NOTIFICATION_API_KEY)%'
+                    User-Agent: 'Denis Internship'
+                max_redirects: 5
+                base_uri: '%env(SMS_NOTIFICATION_BASE_URI)%'
+                timeout: 30
+                max_duration: 120
+                retry_failed:
```
Comment (ovidiuenache) [inline config/packages/framework.yaml:40]: no need for this

File: migrations/Version20220404194440.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE TABLE messenger_messages (id BIGINT AUTO_INCREMENT NOT NULL, body LONGTEXT NOT NULL, headers LONGTEXT NOT NULL, queue_name VARCHAR(190) NOT NULL, created_at DATETIME NOT NULL, available_at DATETIME NOT NULL, delivered_at DATETIME DEFAULT NULL, INDEX IDX_75EA56E0FB7336F0 (queue_name), INDEX IDX_75EA56E0E3BD61CE (available_at), INDEX IDX_75EA56E016BA31DB (delivered_at), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8mb4 COLLATE `utf8mb4_unicode_ci` ENGINE = InnoDB');
```
Comment (ovidiuenache) [inline migrations/Version20220404194440.php:23]: Why did you remove a migration in this PR?

**Review by ovidiuenache — CHANGES_REQUESTED** [review]
Please address the feedback above

File: migrations/Version20220404194440.php (hunk 2 above)
Comment (DenisCrasnean) [reply migrations/Version20220404194440.php:23]: Because I did a git pull origin master and I had the base migrations here for all of the Domain Models. In this pull request I only need the changes which I did on this branch.
